package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-odata-es5",
-  "version": "0.9.0",
+  "version": "0.9.1",
```

Comment:
Can you revert this back to the old value.

The automatic release process takes care of this.

And I think I will create version 0.10.0